### DISCUSSION TITLE
`dhw/start[2]` mix-up? (3.7.0)

### DISCUSSION THREAD

**Discussion by MarkusMit** [emsesp/EMS-ESP32#2217] [General]
Hi. Today I upgraded to `3.7.0` and so far it is all looking good, but just now I noticed that there might have happened a mix-up of entities with the support for multiple hot water circuits: The number of starts show "115" for circuit2 and "0" for circuit1 although our heat pump only has one hot water circuit:
![grafik](https://github.com/user-attachments/assets/479d64d4-c8ba-47d6-9db9-3c3219a24683)

I know, I should first upgrade to the latest dev version, but I'm now away from the location until late friday evening, and I don't like to do firmware upgrades without the option of physical intervention. And since I couldn't find anything about it, I just thought I let you know.

**Comment by MichaelDvP** [comment]
This is bad naming. It's not circuit 2, it's a second counterfor circuit 1 in a different telegram. 
dhw/starts are in e9 published from burners and heatpumps, dhw/starts2 are from 0x495 and only sent from heatpumps.
I dont have a heatpump, any suggestions for right naming?

**Reply by MarkusMit** [reply]
Ah, okay. Then at least the (german) name/description has to be adopted, because "Kreis 2" explicitly says "circuit 2."

Maybe add some "burner/HP" and "HP" to the descriptions, and rename "dhw/starts2" to "dhw/startshp"? 
But I'm sure somebody more familiar with the telegrams/bus than me, can come up with better names... ;-)

**Comment by MarkusMit** [comment]
[Resolved] It seems something went wrong when updating last week. I did upload the `3.7.0` firmware, but it it was still showing a `3.7.0-dev` version I had installed months ago. 
Now I installed `3.7.1-dev.10` and the start properties are again as expected.
